tomcat-gcp-lib/pom.xml

```diff
@@ -21,14 +21,50 @@
     <dependency>
       <groupId>com.google.cloud</groupId>
       <artifactId>google-cloud-datastore</artifactId>
-      <version>1.2.0</version>
+      <version>1.2.1</version>
     </dependency>
```

Comment:
Maven can easily deal with this situation. I think exclusions are used more when you really need them to fix some clash.